### DISCUSSION TITLE
Change Jansson Server main

### DISCUSSION THREAD

**Discussion by BainMcKay** [JanssenProject/jans#8537] [Q&A] [answered]
I cloaned my Janssen Serer, in DigitalOcean, to new Domain (create new VM from snapshot). I need to change the domain name. I did some edits. it continues to reference the old domain. Is there a Json I need to edit. Or do I need to rebuild the Janeen  server from scratch?

**Comment by ossdhaval** [comment]
Hey @BainMcKay I don't think changing the domain name is possible after the installation. Marking @yuriyz to get a confirmation.

**Comment by yuriyz** [comment, marked as answer]
That's right, domain name change is not possible after installation. Clean approach is to re-install with new domain.

**Comment by BainMcKay** [comment]
Thank you for your responseI thought that might be the case, so I rebuilt it. To transfer the data, I know we can export client as json thru TUI. How can we import client json thru TUI?BainBain McKay CEO & Chief ScientistKayvium ***@***.***(613)668-9979                   ___________________                          KAYVIUMBeyond Search - Intelligent Applications that work the way you think                   ___________________THE CONTENTS OF THIS EMAIL ARE COPYRIGHT KAYVIUM CORPORATION 2005-2024 - ALL RIGHTS RESERVED. THIS EMAIL IS NOT TO BE FORWARDED OR SHARED WITHOUT THE PERMISSION OF KAYVIUM CORPORATIONSent from my iPhoneOn May 21, 2024, at 3:57 AM, Dhaval D ***@***.***> wrote:﻿
Hey @BainMcKay I don't think changing the domain name is possible after the installation. Marking @yuriyz to get a confirmation.
